bundle/schema/openapi.go

```diff
@@ -227,14 +240,19 @@ func (reader *OpenapiReader) ResourcesDocs() (*Docs, error) {
 	if err != nil {
 		return nil, err
 	}
+	modelServingDocs, err := reader.modelServingDocs()
+	if err != nil {
+		return nil, err
+	}
 
 	return &Docs{
 		Description: "Collection of Databricks resources to deploy.",
 		Properties: map[string]*Docs{
-			"jobs":        jobsDocs,
-			"pipelines":   pipelinesDocs,
-			"experiments": experimentsDocs,
-			"models":      modelsDocs,
+			"jobs":         jobsDocs,
+			"pipelines":    pipelinesDocs,
+			"experiments":  experimentsDocs,
+			"models":       modelsDocs,
+			"modelServing": modelServingDocs,
```

Comment:
This doesn't match the case in the configuration.

Please add an integration test under `bundle/tests` that defines a serving endpoint, and then manually use the JSON schema to verify that the validation works correctly. Also see https://docs.databricks.com/en/dev-tools/bundles/work-tasks.html#step-2-create-bundle-settings-files-for-the-project for how to set this up manually.